src/commands/database/list.mjs

```diff
@@ -68,11 +57,24 @@ async function listDatabasesWithSecret(argv) {
   }
 }
 
-async function listDatabases(argv) {
+export async function listDatabases(argv) {
+  let databases;
+  if (argv.secret) {
+    databases = await listDatabasesWithSecret(argv);
+  } else {
+    databases = await listDatabasesWithAccountAPI(argv);
+  }
+  return databases;
+}
```

Comment:
refactored this so listDatabases is a generic API layer, and the rendering is done in doListDatabases, which is the command handler